files/build_templates/per_namespace/bgp.service.j2

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=BGP container
+{% if multi_instance == 'true' %}
+Requires=database@%i.service
+After=database@%i.service
+{% endif %}
```

Comment:
Since even without multi-instance, bgp requires database.service, this could further be simplified to:

```suggestion
Requires=database{% if multi_instance == 'true' %}@%i{% endif %}.service
After=database{% if multi_instance == 'true' %}@%i{% endif %}.service
```

Reply:
Fixed as per comment.